src/api/pokemon.spec.js

```diff
@@ -0,0 +1,17 @@
+
+// example result of calling getPokemonById(1):
+// { id: 1, name: 'Bulbasaur', photoUrl: 'someurl', types: [ { id: 3, name: 'grass' }, { id: 6, name: 'poison' } ] }
+// example result of calling getTypeById(3):
+// { id: 3, name: 'grass' }
+
+import { getPokemonById, getTypeById } from "../api/pokemons.js";
+
+describe("Get pokemon details from API", function(){
+    test("Should return pokemon details based on its id.", function() {
```

Comment:
Because getPokemonById is `async` function, this testing function should be async, too (also, arrow notation is cool!)
```suggestion
    test("Should return pokemon details based on its id.", async () => {
```